pkg/lib/authenticationflow/declarative/intent_signup_flow_step_create_authenticator.go

```diff
@@ -45,6 +46,7 @@ type IntentSignupFlowStepCreateAuthenticator struct {
 
 	Options                          []CreateAuthenticatorOptionInternal `json:"options,omitempty"`
 	ShowUntilAMRConstraintsFulfilled bool                                `json:"show_until_amr_constraints_fulfilled,omitempty"`
+	CannotBeSkipped                  bool                                `json:"cannot_be_skipped,omitempty"`
```

Comment:
Originally, under what situation this step could be skipped?

Reply:
In signup flow, create authenticator can always be skipped if `Options` is empty.

Reply:
> In signup flow, create authenticator can always be skipped if Options is empty.

In non-customized authflow, how could that happen?